src/scss/ui/_tag.scss

```diff
@@ -0,0 +1,68 @@
+.tag-container {
+    display: flex;
+    flex-wrap: wrap;
+
+    .tag {
+        background-color: $tag-bg-color;
+        height: 23px;
+        border: 1px solid gray;
+        padding: 0 .2rem;
+        margin: 0 1rem;
+        border-radius: 5px;
+    }
+
+    .cross-icon-tag {
+        display: flex;
+        align-items: center;
+        gap: $tag-gap;
+
+        .icon {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            padding: 0;
+            margin: 0;
+            height: 16px;
+            font-size: 14px;
+            transition: .5 all;
+
+            &:hover {
+                background-color: $tag-icon-hover-color;
+                cursor: pointer;
+            }
+        }
+    }
+
+    .number {
```

Comment:
it doesn't have to be number, we have `badge` component and it should be here https://preview.tabler.io/badges.html

Reply:
So do you mean I should copy-paste the whole badge and put it in the tag.html instead of creating a new tag?
<img width="273" alt="image" src="https://github.com/tabler/tabler/assets/86418083/57c58ade-f1f1-4a04-b7ad-3ff546c529e9">

Reply:
you can use just `{% include ui/badge.html %}` to tag, the same way like in button component